mobile/build.gradle

```diff
@@ -18,6 +18,7 @@ android {
     }
 
     dependencies {
-        compile 'com.novoda:accessibilitools:1.4.0'
+        implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
What's this required for?

Reply:
The @DrawableRes annotation in ToggleTalkbackTileService, lines 23/27 for the import, line 88 for the usage. It's the same one you've been using, it just moved to a new repo that's not available without the new androidx / support-v4 library.